app/controllers/room_guests_controller.rb

```diff
@@ -0,0 +1,13 @@
+class RoomGuestsController < ApplicationController
+  def create
+    @room_guest = RoomGuest.create!(room_guest_params)
+    redirect_to "/guests/#{@guest.id}"
+  end
+
+
+  private
+
+  def room_guest_params
+    params.permit(:room_id, :guest_id)
+  end
+end
```

Comment:
I like how you are using strong params here. One thing to be careful of, is that a a`!` will render a gross error if for whatever reason a `RoomGuest` isn't able to get created. This could be for multiple reasons.. maybe a user passes in an invalid room id or guest id. Sometimes, I think people think that the bang will force a save, but really, it forces an error IF the object can't be saved to the db.